googleSheetsService.py

```diff
@@ -0,0 +1,6 @@
+import gspread
+
+gc = gspread.service_account()
+sheet = gc.open("UsersTests")
+
+wks = sheet.worksheet("Sheet1")
```

Comment:
make here a function which will get user object and will add it to sheets, whole logic should be covered here